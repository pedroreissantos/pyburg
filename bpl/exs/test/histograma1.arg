0 19 2 200
