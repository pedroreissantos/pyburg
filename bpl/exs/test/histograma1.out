24 de 0 a 1: XXXXXXXXXXXXXXXXXXXXXXXX
19 de 2 a 3: XXXXXXXXXXXXXXXXXXX
19 de 4 a 5: XXXXXXXXXXXXXXXXXXX
12 de 6 a 7: XXXXXXXXXXXX
26 de 8 a 9: XXXXXXXXXXXXXXXXXXXXXXXXXX
23 de 10 a 11: XXXXXXXXXXXXXXXXXXXXXXX
21 de 12 a 13: XXXXXXXXXXXXXXXXXXXXX
17 de 14 a 15: XXXXXXXXXXXXXXXXX
15 de 16 a 17: XXXXXXXXXXXXXXX
24 de 18 a 19: XXXXXXXXXXXXXXXXXXXXXXXX
